src/components/draft-js-editor/DraftJSEditor.js

```diff
@@ -19,6 +19,7 @@ const OptionalFormattedMessage = () => (
     </span>
 );
 type Props = {
+    description: React.Node,
```

Comment:
Should this be an optional parameter to avoid a breaking change for other consuming components?